### ISSUE TITLE
data type in augmentation

### ISSUE THREAD

**Issue by jizhang02** [ZFTurbo/volumentations#32]
Hello,

I'm curious whether the data type is limited to uint8, ranging from 0-255. In certain situations, such as medical imaging, the data type might be float32. Converting it to uint8 could result in a loss of information.

When I use `import torchvision.transforms`, it also only support uint8.

**Comment by ZFTurbo** [comment]
I think it's because transforms came from 2D images, where 0-255 range is typically used for RGB images. May be volumentations can be updated to work with float32.
